main.py

```diff
@@ -172,17 +157,55 @@ def watermark_path(ratio):
     return file_path
 
 
-def collect_timestamp(message):
+def set_output_path(input_path):
+    output_path = input("Where would you like the output saved?\n"
+                        "e.g. 'home/Dave/Desktop/output.mp4'\n"
+                        "Providing only a filename - e.g. 'file.mp4' - will export to the directory the script is in.\n")
+
+    if output_path:
+        return output_path
+
+    else:
+        filename, extension = os.path.splitext(input_path)
+        default_output_path = "{0}_access_copy{1}".format(filename, extension)
+
+        print("No output file path given!\n"
+              "Saving as default: {0}".format(default_output_path))
+
+        return default_output_path
+
+
+def set_trim_status():
+    trim = input("Do you want to trim this file? ('y'/'n')\n")
+
+    if not (trim == "y" or trim == "n"):
+        print("Invalid input! I'll ask again")
+        set_trim_status()
```

Comment:
Using recursion here, as already for `set_watermark_status`, and below for `set_watermark_status`. You may wish a different behaviour.